src/Infrastructure/Persistence/ParticipantDynamoDbRepository.cs

```diff
@@ -41,15 +41,11 @@ public async Task<ParticipantDetails> GetParticipantDetailsAsync(string particip
                 _config);
         }
 
-        public async Task<ParticipantDetails> GetParticipantDetailsByEmailAsync(string email)
+        private async Task<ParticipantDetails> GetParticipantDetailsByFilterAsync(
```

Comment:
Make it explicit that this is a scan operation.
`ScanForParticipantDetailsWithFilterAsync`